test/integration/taxon_browsing_test.rb

```diff
@@ -152,6 +153,34 @@ def and_i_can_see_links_to_the_child_taxons_in_an_accordion
     end
   end
 
+  def and_the_accordion_has_tracking_attributes
+    tracked_links = page.all(:css, "a[data-track-category='navAccordionLinkClicked']")
+    tracked_links.size.must_equal @child_taxons.size * search_results.size
+
+    tracked_links.each_with_index do |link, index|
+      section_number = (index / search_results.size).floor + 1
+      item_number = index % search_results.size + 1
+
+      link[:'data-track-action'].must_equal "#{section_number}.#{item_number}"
+      link[:'data-track-label'].must_equal "#{search_results[item_number - 1]['link']}"
+      link[:'data-track-value'].must_equal "#{search_results.size}"
+      link[:'data-track-dimension'].must_equal "#{search_results[item_number - 1]['title']}"
+      link[:'data-track-dimension-index'].must_equal '29'
+      link[:'data-module'].must_equal 'track-click'
+    end
+
+    # Test an example free from logic
+    assert page.has_css?(
```

Comment:
Hmm. The only issue with this is that the failure message won't be very meaningful - it'll just say "Can't find element with <all this stuff>". If there's an easy way of grabbing the relevant element and asserting all the properties individually then that might be better. Otherwise, I think this is good. 👍